l10n_it_vat_statement_communication/migrations/12.0.1.1.0/post-migrate.py

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2019, Link IT srl, Italy. All rights reserved.
+#
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+def migrate(cr, version):
+    if not version:
+        _logger.warning("""
+            Non è presente alcuna versione precedente del modulo.
+            Skippo la migration.
+            """)
+
+        return
+
+    _logger.info("Imposto il valore di default sul nuovo campo delle company")
+    cr.execute("""
+        UPDATE res_company
+        SET vsc_supply_code = 'IVP18';
+    """)
+
+    _logger.info("Ho terminato l'esecuzione della migration del modulo.")
```

Comment:
Anche queste due, se compaiono nel .pot andrebbero in inglese.

Reply:
questa cosa mi sfugge... 
nel pot li devo aggiungere io?

Reply:
> questa cosa mi sfugge...
> nel pot li devo aggiungere io?

In genere il .pot viene generato automaticamente, ma non so se Odoo contempli i file di migrazione nell'analisi delle stringhe.

Qualcuno ne sa di più? @OCA/local-italy-developers

Reply:
Se non viene usato `_(`, la stringa non è traducibile e quindi non compare nel .pot

Reply:
Come esportazione esplicita ok, intendevo esportazioni implicite tipo attributi string e help.

Reply:
Che io sappia `_logger.info` non genera implicitamente il termine traducibile. Tra l'altro l'ho sempre visto solo in inglese

Reply:
Ok, magari potremmo fissare delle linee guida per avere uniformità in casi come questo.

Le due alternative che mi vengono in mente sono:

1) Impostiamo come traducibili e quindi compaiono nel .pot.
2) Le lasciamo in inglese.

Reply:
per ora le ho messe in inglese, così non si sbaglia :)